packages/remote-feature-flag-controller/CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.3.0]
+
 ### Changed
 
+- fix(3742): improve user segmentation with BigInt-based random generation ([#5110](https://github.com/MetaMask/core/pull/5110))
```

Comment:
Could you remove the `fix(3742)` prefixes here? We don't use category prefixes, since we have headers instead. And it's a bit confusing to have this described as a fix given that it hasn't been categorized as such

Reply:
Yes definitely ! good catch ! removed in 810d7ee6f628fb19bbd48c5e5fcb3dca168e3294